src/run_cmd.js

```diff
@@ -37,7 +37,9 @@ export default function runCmd (cwd, argv) {
   }, function (err, statuses) {
     if (err) throw err
     const info = scripts.map((script, i) =>
-      script + ' exited with status ' + statuses[i]
+      pkg.name + ': '
+      + script + ' exited with status ' + statuses[i] + ' '
+      + 'in directory ' + cwd
```

Comment:
cwd will always be `process.cwd()`, which I think is expected. I don't think we have to log that.